traits-stubs/traits_stubs_tests/examples/Float.py

```diff
@@ -11,6 +11,18 @@
 from traits.api import Float, HasTraits
 
 
+class HasIndex:
```

Comment:
Dumb question (and probably not for this PR in any case): do we have a test for this sort of thing in traits test suite?

Reply:
Yes, I'm fairly sure we do. :-) I'll check.

Reply:
Int: https://github.com/enthought/traits/blob/80294cf76e56529587d6007f0f0c69fe8dcfe352/traits/tests/test_integer.py#L32-L34

Float: https://github.com/enthought/traits/blob/80294cf76e56529587d6007f0f0c69fe8dcfe352/traits/tests/test_float.py#L21-L44